src/web-interface/src/messages.odvd

```diff
@@ -1,17 +1,47 @@
+/********************************************************/
+/** V2V Protocol specific messages **********************/
+/********************************************************/
 message AnnouncePresence [id = 1001] {
   string vehicleIp [id = 1];
-  uint16 activePort [id = 2];
   string groupId [id = 3];
 }
 
 message FollowRequest [id = 1002] {
+  uint8 status [id = 1];
 }
 
 message FollowResponse [id = 1003] {
-  string ntpServerIp [id = 1];
+  uint8 status [id = 1];
 }
 
 message StopFollow [id = 1004] {
+  float status [id = 1];
+}
+
+message LeaderStatus [id = 2001] {
+  uint32 timestamp [id = 1];
+  float speed [id = 2];
+  float steeringAngle [id = 3];
+  uint8 distanceTraveled [id = 4];
+}
+
+message FollowerStatus [id = 3001] {
+  uint8 status [id = 1];
+}
+
+
+/********************************************************/
+/** Internal Messages ***********************************/
+/********************************************************/
+
+message Role [id = 2203] {
+  uint8 current [id = 1];
+}
+
+message opendlv.proxy.AccelerationReading [id = 1030] {
```

Comment:
My bad, shouldn't be included (the .AccelerationReading)

Reply:
Are we not using this for later?

Reply:
Not that I'm aware of, but I could be wrong ofc